static/versionDetails.json

```diff
@@ -300,7 +300,7 @@
     },
     {
       "flag": "disableS3",
-      "description": "Migrate application archives and support bundles from S3 and use a local volume in the kotsadm statefulset instead. The migration process is irreversible and will replace the kotsadm deployment with a statefulset. Defaults to 'false'",
+      "description": "Migrate application archives and support bundles from S3 and use a local volume in the kotsadm statefulset instead. The migration process is irreversible and will replace the kotsadm deployment with a statefulset. While this reduces KOTS dependency on an object store for the application archive and support bundle features, an object store is still required to deploy KOTS. Defaults to 'false'",
```

Comment:
Good grammar catch; I added a " ' " to KOTS (shouldn't need the additional "s" I think)

Reply:
Depends on the style guide. AP is the only that says you don't need the "s" after the apostrophe, and they're thinking of changing it. But lots of people write it that way unknowingly. Just a fun fact. https://twitter.com/APStylebook/status/1186716382785949696

Reply:
Gotcha;  I'm ok with either way.
The period at the end of the line could also  be in the quote I think, though only 1 other line of the table (version flag) ends with a period; the rest do not end with a period so the style is overall a bit inconsistent.  Whichever you think is best works for me

Reply:
If it was just a single phrase like most of the others, we could keep it without a period. But since it's multiple sentences with periods, I think it's better to end with a period too.